src/css/jwplayer/imports/reset.less

```diff
@@ -1,4 +1,10 @@
 .jw-reset {
+    &:extend(.jw-reset-text);
+    text-align: left;
+    direction: ltr;
+}
+
+.jw-reset-text {
```

Comment:
Check how this change is output. You don't want two rules that repeat all the same settings:

bad:
```css
.jw-reset {
  a;
  b;
  c;
  d;
}
.jw-reset-text {
  a;
  b;
  c;
}
```

You want the repeated settings under one set, and the additional ones added on:
good:
```css
.jw-reset {
  d;
}
.jw-reset, .jw-reset-text {
  a;
  b;
  c;
}
```